scripts/luna/LUNA_train.py

```diff
@@ -161,11 +162,19 @@ def get_parser():
         parser.error('Required to set --config_file')
 
     if not args.train_path:
-        parser.error('Required to set --working_path')
+        parser.error('Required to set --train_path')
```

Comment:
This is a small thing, but you don't need this if-statement. You can give "required=True" in creating argparse. See https://docs.python.org/3/library/argparse.html#required